### PR TITLE
Add options for Tagging to support middleware

### PR DESCRIPTION
**Pull request by PanSpagetka**
Add options for Tagging to support middleware and to be able to enable better customization

### CODE REVIEW

**Comment by skateman** [comment]
@miq-bot add_reviewer @Hyperkid123 
@miq-bot add_reviewer @karelhala

File: src/tagging/components/InnerComponents/TagModifier.js [hunk 1]
```diff
@@ -3,24 +3,29 @@ import PropTypes from 'prop-types';
 import { Row, Col, Form } from 'patternfly-react';
 import { __ } from '../../../global-functions';
 
-const TagModifier = ({ header, children }) => (
-  <React.Fragment>
+const TagModifier = ({ header, hideHeader, children }) => {
+  const h = hideHeader ? null : (
```
Comment (karelhala) [inline src/tagging/components/InnerComponents/TagModifier.js]: No need to assign to variable right?
```JSX
const TagModifier = ({ header, hideHeader, children }) => (
  <React.Fragment>
      { !hideHeader && <Row>
           <Col lg={12}>
            <h2>{header}</h2>
           </Col>
      </Row> }
    <Form horizontal>{children}</Form>
  </React.Fragment>
)
```

File: src/tagging/components/InnerComponents/TagView.js [hunk 2]
```diff
@@ -19,13 +19,15 @@ class TagView extends React.Component {
 
   render() {
     const assignedTags = [...this.props.assignedTags];
+    const h = this.props.hideHeader ? null : (
```
Comment (karelhala) [inline src/tagging/components/InnerComponents/TagView.js]: No need to assign to variable use same principle as above.

**Pull request merged**